timesketch/lib/analyzers/contrib/hashlookup_analyzer.py

```diff
@@ -53,25 +54,35 @@ def get_kwargs():
     def get_hash_info(self, hash_value):
         """Search event on Hashlookup.
 
+        Args:
+            hash_value:  hash value that will be check
+                         if it's a known one on hashlookup.
+
         Returns:
-            JSON of Hashlookup's results
+            JSON of Hashlookup's results.
         """
         results = requests.get(f"{self.hashlookup_url}sha256/{hash_value}")
 
-        if not "message" in results.json() and results.status_code != 200:
+        result_loc = results.json()
+        if not "message" in result_loc and results.status_code != 200:
             logger.error("Error with Hashlookup url")
             return []
-        if "message" in results.json():
+        if "message" in result_loc:
```

Comment:
What does this mean (`message` present in `result_loc`) for the hashlookup API? Can you add a comment?